rust/libnewsboat/src/human_panic.rs

```diff
@@ -71,23 +71,22 @@ use backtrace::Backtrace;
 use std::io::{self, stderr, BufWriter, Write};
 use std::panic;
 
-#[allow(deprecated)] // Change to PanicHookInfo when our MSRV has increased to 1.81.0 or later
-use std::panic::PanicInfo;
+use std::panic::PanicHookInfo;
 
 /// Sets up a panic hook with a user-friendly message.
 ///
 /// See module description for details.
 pub fn setup() {
     if ::std::env::var("RUST_BACKTRACE").is_err() {
         #[allow(deprecated)]
```

Comment:
Are these `#[allow(deprecated)]` attributes further into this file still necessary?

Reply:
Oops, I missed them >_< Fixed now. Thanks!